docs-dev/module.rst

```diff
@@ -0,0 +1,22 @@
+``vicar`` Module
+================
+
+.. automodule:: vicar
+    :member-order: groupwise
+    :members:
+    :undoc-members:
+    :special-members:
+    :private-members:
+    :exclude-members: __dict__, __hash__, __module__, __weakref__
```

Comment:
These excludes need to be added to the main `docs/module.rst` as well. Right now things like `__dict__` and `__weakref__` are showing up in the main documentation.

Reply:
Done